packages/components/psammead-navigation/README.md

```diff
@@ -175,13 +180,17 @@ import React from 'react';
 import { CanonicalMenuButton } from '@bbc/psammead-navigation/dropdown';
 import { latin } from '@bbc/gel-foundations/scripts';
 
-<CanonicalMenuButton 
-    announcedText="Menu"
-    isOpen={true}
-    onOpen={() => { console.log("Handle open action"); }}
-    onClose={() => { console.log("Handle close action"); }} 
-    script={latin}
-/>
+<CanonicalMenuButton
+  announcedText="Menu"
+  isOpen={true}
+  onOpen={() => {
+    console.log('Handle open action');
```

Comment:
Do you need this for testing atm?

Reply:
I think this was just in the README to show that you need to pass a function in there, prettier just reformatted the file when I updated the README